README.md

```diff
@@ -0,0 +1,35 @@
+# sourir
+
+sourir is better than [RIR](https://github.com/reactorlabs/rir)
+
+## Howto
+
+Install [Opam](https://opam.ocaml.org/doc/Install.html)
+
+    opam install ocaml.4.04.0
+    git clone https://github.com/reactorlabs/sourir
+    cd sourir
+    make sourir
+    ./sourir test.sou
+
+## Syntax / Semantics
+
+The language is still in flux and there is no written syntax or semantics. See
+parser.mly or the testcases in tests.ml to get an idea of the syntax. See
```

Comment:
ok, or we just get rid of the assembler, or is there a good reason to keep it? I already converted some of the examples, since I broke annotations in the parser and didn't bother to fix.